### PR TITLE
enrique/185733446-update-change-healthcare-trading-partner-id-lookup-to-payer-finder-v1

### PR DESCRIPTION
**Pull request by Btace13**
Updated Endpoint for trading_partner request.

- updated endpoint on trading_partner
- updated params on trading_partner to only return Eligibility service types
- updated mapping on trading_partner_data for new response from endpoint
- removed medical_eligibility_enabled from trading_partners_data
- updated tests

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+# [4.13.2] - 2022-08-24
```
Comment (vechema) [inline CHANGELOG.md:8]: At the bottom of the file, need a line like but with the new version

`[4.13.1]: https://github.com/WeInfuse/change_health/compare/v4.13.0...v4.13.1`

**Review by vechema — CHANGES_REQUESTED** [review]
These changes are breaking backwards compatibility. You need to either go to version v5.0.0 or maintain backwards compatibility

**Review by vechema — APPROVED** [review]
lgtm

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,20 @@ All notable changes to this project will be documented in this file.
...
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+# [5.0.0] - 2022-08-24
+
+### Changed
+
+Endpoint for trading_partner request has changed.
+
+* **breaking:** removed medical_eligibility_enabled from trading_partners_data
+* **breaking:** removed medical_eligibility_service_id from trading_partner_data
```
Comment (acrodriguez) [inline CHANGELOG.md:15]:
```suggestion
* **breaking:** removed medical_eligibility_service_id from trading_partner_data
* **breaking:** removed medical_eligibility_service from trading_partner_data
```

**Review by acrodriguez — CHANGES_REQUESTED** [review]
Merge conflicts and one change addition to CHANGELOG

**Review by acrodriguez — APPROVED** [review]

**Pull request merged**